### PR TITLE
Add dual arm models

### PR DESCRIPTION
**Pull request by JoseBarreiros-TRI**
This includes directives for dual arms for 2d and 3d with and without ground and a model with added collision geometries of the jaco.

### CODE REVIEW

File: models/q_sys/dual_jaco_box_no_ground.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+model_directive: package://quasistatic_simulator/dual_jaco_sphere_collision_push_no_ground.yml
```
[the remark this answers is not in the document]
Comment (JoseBarreiros-TRI) [reply models/q_sys/dual_jaco_box_no_ground.yml:1]: there is no purpose for this. proceeded to delete the model.

File: models/q_sys/dual_jaco_box.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    name: jaco_left
+    Kp: [800, 600, 600, 600, 400, 200, 200]
+  -
+    name: jaco_right
+    Kp: [800, 600, 600, 600, 400, 200, 200]
+
+objects:
+  -
+    name: box
+    file: package://quasistatic_simulator/box_10cm.sdf
```
[the remark this answers is not in the document]
Comment (JoseBarreiros-TRI) [reply models/q_sys/dual_jaco_box.yml:13]: fixed

File: models/dual_jaco_sphere_collision_push.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+- add_model:
+    name: jaco_right
+    file: package://quasistatic_simulator/j2s7s300_arm_sphere_collision_v2.urdf
+
+- add_frame:
+    name: world_left_arm_offset
+    X_PF:
+        base_frame: world
+        translation: [0, 0.406, 0]
```
[the remark this answers is not in the document]
Comment (JoseBarreiros-TRI) [reply models/dual_jaco_sphere_collision_push.yml:17]: this is just a placeholder, will update it once the hw setup is ready.

**Comment by JoseBarreiros-TRI** [comment]
first pass done

File: models/q_sys/dual_jaco_box_no_ground.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (aykut-tri) [reply models/q_sys/dual_jaco_box_no_ground.yml]: cool, thanks

File: models/q_sys/dual_jaco_box.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (aykut-tri) [reply models/q_sys/dual_jaco_box.yml]: thank you!

File: models/dual_jaco_sphere_collision_push.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (aykut-tri) [reply models/dual_jaco_sphere_collision_push.yml:17]: gotcha, thanks for the clarification.

**Pull request merged**